pkg/occlient/fakeclient.go

```diff
@@ -65,7 +65,7 @@ func FakeNew() (*Client, *FakeClientset) {
 	client.buildClient = fkclientset.BuildClientset.BuildV1()
 
 	if os.Getenv("KUBERNETES") != "true" {
-		client.GetKubeClient().SetDiscoveryInterface(fakeDiscoveryWithRoute)
+		//client.GetKubeClient().SetDiscoveryInterface(fakeDiscoveryWithRoute)
```

Comment:
This has been commented out because it gets overriden by the next statement.

Reply:
We can remove the comments for now

Reply:
Are you suggesting I keep it as it is for now? I can do that, I have issue #4773  open, I can fix this when I work on that issue.

Reply:
No, we should remove it for now